docs/index.md

```diff
@@ -100,9 +93,6 @@ for examples of how to start using databases together with SQLAlchemy core expre
 [alembic]: https://alembic.sqlalchemy.org/en/latest/
 [psycopg2]: https://www.psycopg.org/
 [pymysql]: https://github.com/PyMySQL/PyMySQL
-[asyncpg]: https://github.com/MagicStack/asyncpg
```

Comment:
I would leave the links and add new links to rest of the drivers. It should allow people to choose a driver they want, especially for backends that support more than one (MySQL and postgres)